client/checkout/blocks/style.scss

```diff
@@ -2,6 +2,10 @@
 	margin-bottom: 2rem;
 }
 
+.wcpay-card-mounted {
+	background-color: #fff;
+}
+
```

Comment:
Doen;t look like a TS thingy =) What's the background for introducing this change?